src/main/kotlin/com/hibob/feedback/resource/FeedbackResource.kt

```diff
@@ -0,0 +1,56 @@
+package com.hibob.feedback.resource
+
+import com.hibob.academy.service.SessionService.Companion.SECRET_KEY
+import com.hibob.feedback.dao.*
+import io.jsonwebtoken.Jwts
+import jakarta.ws.rs.*
+import jakarta.ws.rs.container.ContainerRequestContext
+import jakarta.ws.rs.core.Response
+import org.springframework.stereotype.Component
+import org.springframework.web.bind.annotation.RequestBody
+import java.util.*
+import jakarta.ws.rs.core.MediaType
+
+
+@Component
+@Path("/feedback")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+class FeedbackResource(/*feedback Serrvice*/) {
+    companion object {
+        val cookieName = "Jwt"
+    }
+
+    private fun buildActiveUser(requestContext: ContainerRequestContext): ActiveUser{
+        val cookie = requestContext.cookies[cookieName]?.value
+        val claims = Jwts.parserBuilder().setSigningKey(SECRET_KEY).build().parseClaimsJws(cookie).body
+        val companyId = claims["companyId"] as UUID
+        return ActiveUser(claims["employeeId"] as UUID, companyId)
+    }
```

Comment:
i really like your way of thinking! let's do this in the filter, and use request headers to pass the user from the filter to hear. lemme know if that makes sense to you

Reply:
Yes it is
but I dont have filter yet
Should I add it now or should i finish the service and dao first?

Reply:
don't you have the filter from the JWT class?

i would say to remove this for now (even though i really like you direction), and in another PR let's enhance the filter class to add a ActiveUser object to the request header. then we can tweak the resource to fetch it from there. sound good?

Reply:
great

Reply:
preferably in a separate PR